omod/src/main/java/org/openmrs/module/aijar/web/controller/AppointmentController.java

```diff
@@ -0,0 +1,47 @@
+package org.openmrs.module.aijar.web.controller;
+
+import java.util.Date;
+import java.util.List;
+
+import org.apache.commons.lang3.StringUtils;
+import org.openmrs.api.AdministrationService;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.webservices.rest.web.v1_0.controller.BaseRestController;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+
+@Controller
+@RequestMapping("/rest/ugandaemr/appointments")
+public class AppointmentController extends BaseRestController {
+
+    AdministrationService administrationService = Context.getAdministrationService();
+
+    @RequestMapping(method = RequestMethod.GET)
+    public int getNumberOfPatients(
+            @RequestParam(required = true, value = "nextAppointmentDate") Date nextAppointmentDate,
+            @RequestParam(value = "encounterType", defaultValue = "") String encounterType) {
```

Comment:
@reagan-meant rename this parameter to encounterTypeUuid

Reply:
Alright...

Reply:
Couldn't find any point supporting the use of the appended word Uuid...It is usually implied.